### DISCUSSION TITLE
Does eth.getBlock("latest") and eth.getBlock("pending") returns the same block?

### DISCUSSION THREAD

**Discussion by ofarukcaki** [erigontech/erigon#4874] [Q&A]
Hello, I was using a [bsc fork](https://github.com/bnb-chain/bsc/) of the go-ethereum for the BSC network. But in that implementation a full node does not keep/build a pending block and it cannot access the pending transactions. So that eth.getBlock("latest") and eth.getBlock("pending") are both returning the latest block.

That is why I wanted to check out erigon to see if I can access to pending block while running a full node, my question is **can I access to the pending block using erigon?**

**Comment by AskAlexSharov** [comment]
Probably we don’t support “get pending block”. Created issue https://github.com/ledgerwatch/erigon/issues/4876

**Reply by enriavil1** [reply]
When erigon doesn't have a pending block we returned the latest block inside of the execution stage, so through some period of time they will match.
